huntflow_api_client/models/common.py

```diff
@@ -23,58 +23,52 @@ def jsonable_dict(
         include: Optional[Union[AbstractSetIntStr, MappingIntStrAny]] = None,
         exclude: Optional[Union[AbstractSetIntStr, MappingIntStrAny]] = None,
         by_alias: bool = False,
-        skip_defaults: Optional[bool] = None,
         exclude_unset: bool = False,
         exclude_defaults: bool = False,
         exclude_none: bool = False,
-        encoder: Optional[Callable[[Any], Any]] = None,
-        **dumps_kwargs: Any,
+        round_trip: bool = False,
+        warnings: bool = True,
     ) -> Dict[str, Any]:
         params = {
             "include": include,
             "exclude": exclude,
             "by_alias": by_alias,
-            "skip_defaults": skip_defaults,
             "exclude_unset": exclude_unset,
             "exclude_defaults": exclude_defaults,
             "exclude_none": exclude_none,
-            "encoder": encoder,
+            "round_trip": round_trip,
+            "warnings": warnings,
         }
-        return json.loads(self.json(**params, **dumps_kwargs))  # type: ignore
+        return json.loads(self.model_dump_json(**params))  # type: ignore
```

Comment:
You have all possible parameters, so it's not needed to build a dict, just pass all the params to the model_dump_json:
```
model_dump_json(inlude=include, exclude=exclude, ...)
```